indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/PendingTaskBasedWorkerProvisioningStrategy.java

```diff
@@ -144,25 +168,33 @@ public Worker apply(ImmutableWorkerInfo input)
           ),
           workerConfig
       );
+      log.info("Currently provisioning: %d [%s]", currentlyProvisioning.size(), currentlyProvisioning);
       currentlyProvisioning.removeAll(workerNodeIds);
+      log.info(
+          "Currently provisioning without WorkerNodeIds: %d [%s]",
```

Comment:
Collection.toString() writes brackets too. No need for `[]`.

Reply:
Fixed this and similar places in this class